server/chat-plugins/othermetas.ts

```diff
@@ -542,4 +542,57 @@ export const commands: ChatCommands = {
 	crossevolvehelp: [
 		"/crossevo <base pokemon>, <evolved pokemon> - Shows the type and stats for the Cross Evolved Pokemon.",
 	],
+
+	'!showevo': true,
+	showevo(target) {
+		if (!this.runBroadcast()) return;
+		const targetid = toID(target);
+		if (!targetid) return this.parse('/help showevo');
+		const evo = Dex.getSpecies(target);
+		if (!evo.exists) {
+			return this.errorReply(`Error: Pokemon not found.`);
+		}
+		if (!evo.prevo) {
+			return this.errorReply(`Error: Pokemon does not evolve from another Pokemon.`);
+		}
+		const banlist = Dex.getFormat('gen8crossevolution').restricted || [];
+		if (banlist.includes(evo.name)) {
+			this.errorReply(`Warning: ${evo.name} is banned from being Cross Evolved into.`);
+		}
```

Comment:
We've stopped including these in OM commands for the most part because they're more spammy than helpful